xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-JSONRPC_VERSION 11.1.1
+JSONRPC_VERSION 11.2.1
```

Comment:
That needs to be 11.2.0, patch number gets zeroed

Reply:
Fixed.